.gitignore

```diff
@@ -36,6 +36,8 @@ Thumbs.db
 *.tmPreferences.cache
 # Eclipse
 .settings/*
+/.project
+/.buildpath
```

Comment:
Directories should be `dir/*` for consistency.

But overall this list shouldn't extended to much, there are too many specific paths, Eclipse alone can have [many](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore).

Better location would be your local `~/.gitconfig`.

Reply:
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
`.project` and `.buildpath` are files but not folders. There is nothing wrong in having a bit longer list of entries in .gitignore especially when new entries fit well into the existing list. I never add Eclipse configuration files to global gitignore and probably will never do. Please do not make it more difficult for developers to contribute.

Reply:
I never understood why people are so keen on stuffing their project level `.gitignore`'s with all sorts of stuff unrelated to the actual project code, given that every Git repository has an `info/exclude` file that can be used for ignoring user environment specific files and folders.